src/main/java/edu/hawaii/its/api/groupings/MemberResult.java

```diff
@@ -6,6 +6,8 @@ public abstract class MemberResult {
     private String name;
     private String firstName;
     private String lastName;
+    private boolean isOwnerGrouping;
+    private String ownerGroupingPath;
```

Comment:
Based on your UI code,
```
if(owner.name.includes(':')) {
    owner.isOwnerGrouping = true;
    // Name field in owner groupings initially holds the group path
    owner.ownerGroupingPath = owner.name;
    const splitOwnerPath = owner.name.split(':');
    // Isolate the grouping name from the path
    owner.name = splitOwnerPath[splitOwnerPath.length - 1];
}
```
You can delete the isOwnerGrouping and ownerGroupingPath private variables from this class. Objects in Javascript do not require those keys to be already present to set them.

Reply:
I was looking into if you could extend the MemberResult class with something like OwnerMemberResult, but based on the wrapper implementation, it did not make sense.